policyengine-client/src/policyengine/pages/household/accountingTable.jsx

```diff
@@ -7,6 +8,19 @@ import Spinner from "../../general/spinner";
 import { Spacing } from "../../layout/general";
 import { getTranslators } from "../../tools/translation";
 
+const { Panel } = Collapse;
+
+function HouseholdResultsCaveats() {
+    const country = useContext(CountryContext);
+	return (
+		<Collapse style={{marginTop: 15}} defaultActiveKey={["0"]} ghost>
+			<Panel header={<><ExclamationCircleOutlined />  &nbsp; &nbsp;Disclaimer</>} key="1">
+				<p>{`Results are calculated using the OpenFisca-${country.properName} tax-benefit microsimulation model, may not constitute exact tax liabilities or benefit entitlements. See the `}<a href={`https://github.com/PolicyEngine/OpenFisca-${country.properName}`}>repository</a> for more information.</p>
```

Comment:
I don't think we need to get technical here. Could you also default it to uncollapsed? It's important that users see this.
```suggestion
				<p>{`PolicyEngine results may not constitute exact tax liabilities or benefit entitlements.</p>
```